blog/2023-05-05-new-pricing.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: new-pricing-2023-05
+title: 'New pricing is here. Including Pay as you go. Pay only for what you use.'
+authors: [ jcizmar ]
+image: '/img/blog/2023-05-05-new-pricing.png'
+description: 'We are changing our pricing. We are introducing Pay as you go. Pay only for what you use. We are also introducing new metric, which is more fair and easier to understand. We are also introducing new plans for self-hosted instances.'
+tags: [ pricing, pay as you go, self-hosted ]
+---
+
+Based on your feedback, we have updated our pricing. Introducing new pay-as-you-go plans, so you only pay for what you use.
+We also have a fairer and easier-to-understand metric. Additionally, we now offer plans for self-hosted instances.
+
+![New pricing](/img/blog/2023-05-05-new-pricing.png)
+
+<!--truncate-->
+
+## Pay as you go
+
+We are introducing new plans that are based on the number of strings you have in your organization. When you're a free
+user, you don't have to upgrade to a new plan just because of a few new strings. You pay only for what you use. Visit
```

Comment:
```suggestion
We are introducing new plans that are based on the number of strings you have in your organization. When using
the Free plan, you don't have to upgrade to a new one just because you want to add a few new strings. You pay only for what you use. Visit
```